command/agent/consul/connect.go

```diff
@@ -105,12 +105,9 @@ func connectSidecarRegistration(serviceId string, css *structs.ConsulSidecarServ
 		return nil, err
 	}
 
-	return &api.AgentServiceRegistration{
-		Tags:    helper.CopySliceString(css.Tags),
-		Port:    cMapping.Value,
-		Address: cMapping.HostIP,
-		Proxy:   proxy,
-		Checks: api.AgentServiceChecks{
+	var checks api.AgentServiceChecks
```

Comment:
nitpick: since we're building a slice that will always include the alias, i'd golf this down to

```go
checks := api.ServiceChecks {{
 // .. <alias check>
}}
if !css.DisableDefaultCheck {
 checks = append(checks, <tcp check>)
}